panels/user-accounts/org.gnome.controlcenter.user-accounts.gschema.xml.in

```diff
@@ -0,0 +1,12 @@
+<schemalist>
+  <schema id="org.gnome.controlcenter.user-accounts" path="/org/gnome/controlcenter/user-accounts/"
+          gettext-domain="@GETTEXT_PACKAGE@">
+    <key name="facesdir" type="s">
```

Comment:
Possibly, for future proofing, I would suggest allowing more than one directory, make it an array of type `as`.

Reply:
I've updated the code to allow more than one directory.